addons/mail/static/src/discuss/core/common/channel_member_model.js

```diff
@@ -98,7 +101,7 @@ export class ChannelMember extends Record {
     typingTimeoutId;
 
     get name() {
-        return this.channel_id.getPersonaName(this.persona);
+        return this.channel_id.getPersonaName(this.partner_id || this.guest_id);
```

Comment:
As already posted in other PR, `getPersonaName` should be renamed to `getPartnerName` and only be called for partner. None of the conditions inside it make any sense for guest, we actually have to explicitly check for guest to return something different.

Reply:
We could do the rename in another PR? To keep things scoped.